code/online_challenges/src/codechef/CASH/hardcash.cpp

```diff
@@ -0,0 +1,21 @@
+#include<iostream>
+#define ll long long int
+using namespace std;
```

Comment:
Don't use this. Reformat all your files to use clang format with 4 spaces (for C++).